engine/gamesys/proto/gamesys/texture_set_ddf.proto

```diff
@@ -37,19 +37,23 @@ message SpriteGeometry
     required float center_x = 3;
     required float center_y = 4;
 
-    required bool  rotated = 5; // If true, then the image is rotated 90 degrees (ccw)
```

Comment:
Hmm, I thought you were not allowed to change the number id of the message field in Protobuf? E.g. you are supposed to only add new ids, otherwise this will break the deserialization of pre-existing binary messages.

Reply:
This is for the runtime content. We make no guarantuee that they are the same between versions, but I agree, I don't have to rearrange the order.